src/api/EscargotPublic.h

```diff
@@ -982,7 +982,7 @@ class ESCARGOT_EXPORT ValueRef {
     ESCARGOT_TYPEDARRAY_REF_LIST(DEFINE_VALUEREF_IS_AS);
 #undef DEFINE_VALUEREF_IS_AS
 
-    bool toBoolean(ExecutionStateRef* state);
```

Comment:
IMHO we should not change public API if possible
and this requirements can be changed later by ECMAScript spec change